### PR TITLE
Add new bots

### PR DESCRIPTION
**Pull request by oTkPoBeHuE**

### CODE REVIEW

File: src/abstractBot/AbstractBot.js [hunk 1]
```diff
@@ -0,0 +1,52 @@
+import { PromiseQueue } from './PromiseQueue';
+
+export class AbstractBot {
+    constructor(bot) {
+        this.bot = bot;
+        this.promiseQueueMap = new Map();
+    }
+
+    addUser(id) {
```
Comment (ledamint) [inline src/abstractBot/AbstractBot.js]: addChat

File: src/abstractBot/AbstractBot.js [hunk 2]
```diff
@@ -0,0 +1,52 @@
...
+        this.promiseQueueMap.set(id, new PromiseQueue());
+    }
+
+    deleteUser(id) {
```
Comment (ledamint) [inline src/abstractBot/AbstractBot.js]: deleteChat

File: src/vkontakteBot/VkontakteBot.js [hunk 3]
```diff
@@ -0,0 +1,70 @@
+import easyvk from 'easyvk';
+
+function parseMessage(msg) {
+    const peer_id = msg[3];
+    const text = msg[5];
+    return {
+        out: msg[2] & 2,
```
Comment (ledamint) [inline src/vkontakteBot/VkontakteBot.js:7]: Nice code!

File: webpack.config.js [hunk 4]
```diff
@@ -2,7 +2,7 @@ const path = require('path');
 
 module.exports = {
     mode: 'development',
-    entry: './examples/index.js',
+    entry: './examples/VK.js',
```
Comment (ledamint) [inline webpack.config.js]: Please keep './examples/index.js'

**Comment by ledamint** [comment]
Good work

File: src/abstractBot/AbstractBot.js (hunk 1 above)
Comment (oTkPoBeHuE) [reply src/abstractBot/AbstractBot.js]: Fixed

File: src/abstractBot/AbstractBot.js (hunk 2 above)
Comment (oTkPoBeHuE) [reply src/abstractBot/AbstractBot.js]: Fixed

File: src/vkontakteBot/VkontakteBot.js (hunk 3 above)
Comment (oTkPoBeHuE) [reply src/vkontakteBot/VkontakteBot.js:7]: It's magic

File: webpack.config.js (hunk 4 above)
Comment (oTkPoBeHuE) [reply webpack.config.js]: fixed

**Pull request merged**